notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>BTW one thing cool about gradio is it has direct integration with transformers, so you can call <code>gr.Interface.load()</code> here instead of calling pipeline and then Interface. and for more complex second part you could do a bare function and then pass inside Interface.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
By this you mean direct integration with the Hub right? Ie `load` works for models in the hub already. 

I wanted to feature this as well (in fact it was part of my original writeup) but I ended up thinking it would make this recipe too long. Maybe I can add a `TIP` mention in this part to point readers to the `load` feature.

Reply:
Done!

Reply:
apparently in recent version of gradio this was swapped with [`from_pipeline`](https://www.gradio.app/docs/gradio/interface#interface-from_pipeline) which is essentially transformers integration for gradio. it doesn't need to call any API whatsoever it just loads the model and the pipeline into an Interface by inferring their i/o

Reply:
thank you! added a `TIP` highlighting that constructor